### DISCUSSION TITLE
vue-tips/dynamically-change-page-title

### DISCUSSION THREAD

**Discussion by giscus** [Mokkapps/mokkapps-de-discussions#14] [Announcements]
# vue-tips/dynamically-change-page-title

I will show you a few ways how you can easily change the page title when navigating your application

https://mokkapps.de/vue-tips/dynamically-change-page-title

**Comment by OcasoProtal** [comment]
Unfortunately that `pageTitle` in `params` isn't working anymore: https://github.com/vuejs/router/blob/main/packages/router/CHANGELOG.md#414-2022-08-22
